src/domain/settings/regions/new/index.tsx

```diff
@@ -0,0 +1,146 @@
+  })
+  const {
+    formState: { isDirty },
+    handleSubmit,
+    reset,
+  } = form
+  const { mutate } = useAdminCreateRegion()
+  const notification = useNotification()
+
+  const closeAndReset = () => {
+    reset()
+    onClose()
+  }
+
+  const onSubmit = handleSubmit(
+    (data) => {
+      mutate(
+        {
+          name: data.details.name!,
+          countries: data.details.countries.map((c) => c.value),
+          currency_code: data.details.currency_code?.value,
+          fulfillment_providers: data.providers.fulfillment_providers.map(
+            (fp) => fp.value
+          ),
+          payment_providers: data.providers.payment_providers.map(
+            (pp) => pp.value
+          ),
+          tax_rate: data.details.tax_rate!,
+          tax_code: data.details.tax_code || undefined,
+        },
+        {
+          onSuccess: ({ region }) => {
+            notification("Success", "Region created", "success")
+            navigate(`/a/settings/regions/${region.id}`)
+            closeAndReset()
+          },
+          onError: (error) => {
+            notification("Error", getErrorMessage(error), "error")
+          },
+        }
+      )
+    },
+    (errors) => {
+      if (errors.providers && !sections.includes("providers")) {
+        setSections((oldSections) => [...oldSections, "providers"])
+      }
+    }
+  )
+
+  return (
+    <form className="w-full" onSubmit={onSubmit} noValidate>
+      <FocusModal>
+        <FocusModal.Header>
+          <div className="medium:w-8/12 w-full px-8 flex justify-between">
+            <Button
+              size="small"
+              variant="ghost"
+              type="button"
+              onClick={closeAndReset}
+            >
+              <CrossIcon size={20} />
+            </Button>
+            <div className="gap-x-small flex">
+              <Button
+                size="small"
+                variant="primary"
+                disabled={!isDirty}
+                type="submit"
+              >
+                Create region
+              </Button>
+            </div>
+          </div>
+        </FocusModal.Header>
+        <FocusModal.Main className="w-full no-scrollbar flex justify-center">
+          <div className="medium:w-7/12 large:w-6/12 small:w-4/5 max-w-[700px] my-16">
+            <Accordion
+              value={sections}
+              onValueChange={setSections}
+              type="multiple"
+            >
+              <Accordion.Item title="Details" value="details" forceMountContent>
+                <p className="inter-base-regular text-grey-50 mb-xlarge">
+                  Add the region details.
+                </p>
+                <RegionDetailsForm
+                  form={nestedForm(form, "details")}
+                  isCreate
+                />
+              </Accordion.Item>
+              <Accordion.Item
+                title="Providers"
+                value="providers"
+                forceMountContent
+              >
```

Comment:
Question: should we mark this section as required since it's collapsed and the fields inside are required?